model/ActivityMonitoringService.php

```diff
@@ -66,10 +66,10 @@ protected function getLoginQueueLength()
         $actionQueue = $this->getServiceLocator()->get(ActionQueue::SERVICE_ID);
         $activeDeliveryExecution = new GetActiveDeliveryExecution();
 
-        $queue = __('Turned off');
-        $limit = $actionQueue->getLimits($activeDeliveryExecution);
-        if ($limit > 0) {
+        if ($actionQueue->isActionEnabled($activeDeliveryExecution)) {
```

Comment:
Do you think that is a breaking changes?

Reply:
I don't think it's breaking change. Interface of this method didn't change, just internal implementation how do we verify if Login Queue is enabled.

Reply:
Fixed